tests/rptest/tests/random_node_operations_test.py

```diff
@@ -276,6 +276,11 @@ def test_node_operations(self, enable_failures, num_to_upgrade,
         # tp-workload-compaction - topic with compaction
         # tp-workload-fast       - topic with fast partition movements enabled
 
+        def enable_fast_partition_movement():
+            # do not enable fast partition movement with
+            # upgrades as the feature is not enabled
+            return with_tiered_storage and num_to_upgrade == 0
```

Comment:
- Shall this be conditional on the version so we don't forget to remove it?

Reply:
good point, will change this